packages/components/src/components/navigation-item/navigation-item.ts

```diff
@@ -130,21 +145,22 @@ export default class SdNavigationItem extends SolidElement {
       description: this.hasSlotController.test('description'),
       children: this.hasSlotController.test('children')
     };
-    const tag = isLink ? literal`a` : isAccordion ? literal`summary` : literal`button`;
+
     const horizontalPaddingBottom = this.vertical ? 'pb-3' : 'pb-2';
 
     /* eslint-disable lit/no-invalid-html */
     /* eslint-disable lit/binding-positions */
     const root = html`
-      <${tag}
+      <${this.rootTag}
```

Comment:
If possible please do not rewrite this to rootTag. We currently make some replacement here to enable minifying here, but this fails now: https://github.com/solid-design-system/solid/actions/runs/10218039579/job/28273245660?pr=1236#step:7:41